exercises/matching-brackets/README.md

```diff
@@ -1,7 +1,8 @@
-# Bracket Push
+# Matching Brackets
```

Comment:
There was a change in requirements for practice exercises, and this file now needs to be in `.docs/instructions.md` instead of `README.md` (docs: https://github.com/exercism/docs/blob/main/building/tracks/practice-exercises.md#file-docsinstructionsmd)